### PR TITLE
Bookstore: initialize project with components

### PR DESCRIPTION
**Pull request by jssol**
### Project requirements

- [x] Initialise React app.
- [x] The building blocks of your app should be set as re-usable components. Create a directory for your components.
- [x] Add Routes and Links
  - Books - the default view:
    - Should display the list of books (empty at this point but it should be ready for the data) with the Remove button (no functionality yet).
    - Created a component called Book for displaying a single book (with a title and an author) and reuse it in a component that displays a list of books.
  - Should have a form for adding a book (no functionality yet).
    - Create a separate component for this form (with inputs for a title and an author).
  - Categories:
    - Should display a button "Check status" only.
- [x] Styling is not required at this point.

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,11 +1,18 @@
...
-    <>
-      <h1 className="text-8xl font-bold text-red-500 text-center">Works</h1>
-    </>
-  );
-}
+const App = () => (
+  <>
+    <Header />
+    <Routes>
+      <Route path="/" element={<Books />} />
```
Comment (elmar8287) [inline src/App.js]: - [x] Please, consider that in order to **not** show a blank page when visiting the site for the first time, the `Books` component should be the default page - so, you should add `exact` before the path, like `<Route exact path="/">` 👍

File: src/components/Form.js [hunk 2]
```diff
@@ -0,0 +1,12 @@
+const Form = () => (
+  <section className="absolute self-center mx-auto w-10/12 bottom-0 pt-4 mb-12 border-t-2 border-slate-200">
+    <h2 className="text-slate-500 text-xl font-bold mb-2">ADD NEW BOOK</h2>
+    <form className="w-full h-10 flex items-stretch justfiy-between">
+      <input className="w-3/5 p-3 rounded-md text-slate-600 transition-all h-full block outline-1 outline-blue-100 focus:outline-blue-300" type="text" placeholder="Book title" />
+      <input className="w-1/3 p-3 rounded-md text-slate-600 transition-all h-full block ml-4 outline-blue-100 focus:outline-blue-300" type="" placeholder="Category" />
+      <button className="w-1/5 p-3 flex items-center justify-center rounded-md h-full block bg-blue-600 text-white font-semibold ml-4" type="submit">ADD BOOK</button>
+    </form>
```
Comment (elmar8287) [inline src/components/Form.js]: - [x] Please, consider that this component should render a form with inputs for the title and the author of the book. Here you have title and category, not the author - I know you can easily fix it 👍

**Review by elmar8287 — CHANGES_REQUESTED** [review]
Hi, @jssol.

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

- Nice job with styling 👍 
- No linter issues 💯 

## Required Changes ♻️

- [x]  Check the comments under the review.
- [x] Please, try to fix creating a professional README file - your one came from `create-react-app` and needs to be modified. You can use the [MICROVERSE template for the README file](https://github.com/microverseinc/readme-template/blob/master/README.md), as usual 👍 

## Optional suggestions

- N/A

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/Form.js (hunk 2 above)
Comment (jssol) [reply src/components/Form.js]: @elmar8287, I saw on the template it was category, but I can change this no problem

**Review by rotimiazeez — APPROVED** [review]
# Hi @jssol 👍, 

### Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**